### PR TITLE
Add new exercise: rotational-cipher

### LINKED ISSUE #93: Add new exercise: Rotational Cipher

### PR DESCRIPTION
**Pull request by IsaacG**
Closes #93

### CODE REVIEW

File: exercises/practice/rotational-cipher/.docs/instructions.append.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+This exercise requires converting ASCII characters to and from their numeric value.
+GNU awk's [ordchr extension][ordchr] provides this functionality. To add the
+`ord()` and `chr()` functions, load the `ordchr` extension by adding a `@load`
+directive to the top of your file:
+
+```
+@load "ordchr"
+```
+
```
Comment (glennj) [inline exercises/practice/rotational-cipher/.docs/instructions.append.md:13]: Nice!

File: exercises/practice/rotational-cipher/.meta/example.awk [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    FS = ""
+    OFS = ""
+    A = ord("A")
+    a = ord("a")
+}
+
+{
+    out = ""
+    for (i = 1; i <= NF; i++) {
+        if ("A" <= $i && $i <= "Z")
```
Comment (glennj) [inline exercises/practice/rotational-cipher/.meta/example.awk:13]: Ah, my favourite way to write a "between" expression.

File: exercises/practice/rotational-cipher/.meta/example.awk [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+            $i = chr((ord($i) - A + distance) % 26 + A)
```
Comment (glennj) [inline exercises/practice/rotational-cipher/.meta/example.awk:14]: What's `distance`?

File: exercises/practice/rotational-cipher/.meta/example.awk (hunk 2 above)
Comment (IsaacG) [reply exercises/practice/rotational-cipher/.meta/example.awk:13]: No chained operators like Python supports :(

File: exercises/practice/rotational-cipher/.meta/example.awk (hunk 3 above)
Comment (IsaacG) [reply exercises/practice/rotational-cipher/.meta/example.awk:14]: The amount to rotate by. What would be a better name? Or should I have it in the first record?

Comment (glennj) [reply exercises/practice/rotational-cipher/.meta/example.awk:14]: oh, it's the -v variable. That's the somewhat irritating thing about the standalone .awk files. Maybe we could come up with a comment convention so at least the word "distance" (or whatever for other exercises) shows up in search.

**Review by glennj — APPROVED** [review]

**Pull request merged**